client/src/controllers/index.ts

```diff
@@ -34,6 +35,8 @@ export const coreControllerDefinitions: Definition[] = [
   { controllerConstructor: DropdownController, identifier: 'w-dropdown' },
   { controllerConstructor: MessagesController, identifier: 'w-messages' },
   { controllerConstructor: ProgressController, identifier: 'w-progress' },
+  { controllerConstructor: RevealController, identifier: 'w-breadcrumb' },
```

Comment:
Any particular reason why the singular `breadcrumb`?

Reply:
haha, it's always been like that since the initial version @thibaudcolas  built `data-breadcrumb-next` not `data-breadcrumbs-next`.

I am not sure, but have kept with the convention that exists, especially since the root class is `w-breadcrumb` not `w-breadcrumbs`.

If you think it's worth changing, I would 100% prefer the plural form here.

Reply:
Don't really mind either way, but I also would prefer the plural form. If it were only `data-breadcrumb-next` that we're dealing with, I think it's worth renaming, but as we also have `w-breadcrumb` in the CSS, might be better to stick with the existing singular form. If Thibaud weighs in, I'm happy to go with whatever he says 👍

Reply:
I see that our JS events were in the plural form so I think it is reasonable to update this across the board to be plural.

I have done this, will force push and I have included a note in the release notes about the classes change from `w-breadcrumb` to `w-breadcrumbs`.